### PR TITLE
use pkgutil.resolve_name where available

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: requirements/py35.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    ./requirements-compile.sh
+#    ./requirements/compile.sh ./setup.py requirements/requirements.in
```
Comment (graingert) [inline requirements/py35.txt]: bah this doesn't work yet due to https://github.com/jazzband/pip-tools/issues/908

**Comment by adamchainz** [comment]
Thanks for demoing but please remove the python 3.9 changes for now - I'll add it across all my repos automagically when it's ready.

File: requirements/py35.txt (hunk 1 above)
Comment (adamchainz) [reply requirements/py35.txt]: Just put `pkgutil-resolve-name` in the requirements.in file and revert attempts to change the way requirements are compiled (I'd like to keep it uniform across my projects)

**Comment by graingert** [comment]
the `./requirements-compile.sh` script doesn't exist

File: requirements/py35.txt (hunk 1 above)
Comment (graingert) [reply requirements/py35.txt]: the ./requirements-compile.sh script doesn't exist and is missing the path to the requirements.in file

Comment (adamchainz) [reply requirements/py35.txt]: please rebase on master after #202 rewrote the script to Python to avoid those working directory problems

**Comment by graingert** [comment]
> Thanks for demoing but please remove the python 3.9 changes for now - I'll add it across all my repos automagically when it's ready.

done

**Comment by adamchainz** [comment]
Looking good. Would you mind merging the docs PR I made ( https://github.com/graingert/pkgutil-resolve-name/pull/1 ) and releasing a version 1.0.0? I'd rather not depend on something that's "version 0.0.0" since not many Python packages use that and it implies alpha status.

**Comment by graingert** [comment]
> Looking good. Would you mind merging the docs PR I made ( [graingert/pkgutil-resolve-name#1](https://github.com/graingert/pkgutil-resolve-name/pull/1) ) and releasing a version 1.0.0? I'd rather not depend on something that's "version 0.0.0" since not many Python packages use that and it implies alpha status.

Done

**Comment by adamchainz** [comment]
Released in 2.1.0: https://pypi.org/project/patchy/

**Pull request merged**